api/test/integratedModels.test.js

```diff
@@ -0,0 +1,81 @@
+import fetch from "node-fetch";
+
+// This constant should be updated whenever new data (new versions
+// of existing models, or new models) is added to the project.
+const CORRECT_DATA = {
```

Comment:
You are right. Files such as `reactions.tsv` does not exists in the `api` container. Sorry that I was not thinking well. I think it is fair enough to verify the data like that.